### ISSUE TITLE
Advanced mode related bugs:

### ISSUE THREAD

**Issue by Mitsunee** [Mitsunee/Gen3DmgCalc#1]
- *Minor bug*: going back to normal mode should run the calc function to remove the attacker name from the current result
- *Minor bug*: selecting a defender and then changing its type manually does not remove the name in the current result (hidden input for dataset isn't cleared)
- *Enhancement*: no indication for possible HP values is given for the defender (should show 0IV,0EV to 31IV,255EV). Idk if automatic override of the value in the field would be wanted, but it could be made to default to the highest possible value, should one be entered that's even higher. Lower than lowest possible max value is fine as its current HP not max HP
- **CRITICAL**: Advanced mode only uses ATK and DEF, never SpA and SpDef.

**Comment by Mitsunee** [comment]
I might revert a lot of the advanced mode stuff and redo a big part of it. I wanna make the main function call for updates if applicable and I need to change it to actually check that attack type (Physical or Special) before doing so....
TL;DR: Will I ever be happy with how I do things and not redo them 50 times.
